pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -71,6 +71,27 @@ elif command -v dnf >/dev/null 2>&1; then
 		# Workaround for https://github.com/containerd/stargz-snapshotter/issues/340
 		ln -s fusermount3 /usr/bin/fusermount
 	fi
+elif command -v yum >/dev/null 2>&1; then
```

Comment:
Please add a comment line to explain that CentOS 7 support is deprecated

Reply:
Done!